manifests/client.pp

```diff
@@ -100,7 +100,14 @@
 
   if $expire {
     if is_integer($expire) {
-      $env_expire = "KEY_EXPIRE=${expire}"
+      case $openvpn::easyrsa_version {
+        '2.0': {
+          $env_expire = "KEY_EXPIRE=${expire}"
+        }
+        '3.0': {
+          $env_expire = "EASYRSA_CERT_EXPIRE=${expire} EASYRSA_NO_VARS=1"
```

Comment:
I've used EASYRSA_NO_VARS because by default easy-rsa source the "vars" file at startup and then overwrite variable given by shell environment. With EASYRSA_NO_VARS, easy-rsa do not source anything, but because we use ". vars" in exec resource, all vars are already provisionned in shell environment

 https://github.com/OpenVPN/easy-rsa/blob/master/easyrsa3/easyrsa#L2481